authors/apps/articles/views.py

```diff
@@ -16,7 +16,8 @@
 from authors.apps.authentication.models import User
 from .utils import is_article_owner, has_reviewed, round_average
 from .filters import ArticleFilter
-
+from .utils import generate_share_url
+import cloudinary
```

Comment:
I have noticed this cloudinary import is not used in your social share code and it is also a duplicate import in the file. Please remove this.

Reply:
> I have noticed this cloudinary import is not used in your social share code and it is also a duplicate import in the file. Please remove this.

Thanks for pointing it out, I have removed the duplicate import.